nifi-manifest/nifi-runtime-manifest-core/pom.xml

```diff
@@ -44,5 +44,10 @@
             <artifactId>nifi-extension-manifest-parser</artifactId>
             <version>1.16.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
```

Comment:
Is this additional dependency necessary?

Reply:
Yes there is some code in this module that uses an ObjectMapper to serialize the manifest to JSON